src/test/java/racingcar/UnitTest.java

```diff
@@ -0,0 +1,193 @@
+package racingcar;
+
+import camp.nextstep.edu.missionutils.test.NsTest;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static camp.nextstep.edu.missionutils.test.Assertions.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class UnitTest extends NsTest {
+
+    @Test
+    @DisplayName("readInput 테스트")
+    void readInputTest() {
+        String input = "car1,car2,car3\n";
+        System.setIn(new ByteArrayInputStream(input.getBytes()));
+        String result = Application.readInput();
+        assertThat(result).isEqualTo("car1,car2,car3");
+    }
+
+    @Test
+    @DisplayName("splitInput 작동테스트")
+    void splitInputTest() {
+        List<RacingCar> resultCars = Application.splitInput("car1,car2, car3");
+        assertThat(resultCars).extracting("name").containsExactly("car1", "car2", "car3");
+    }
+
+    @Test
+    @DisplayName("splitInput 같은 이름이 있을 경우")
+    void slitInputTest2() {
+        assertThatThrownBy(() -> Application.splitInput("car1,car2,car1"))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("동일한 이름을 가진 차종을 입력하실 수 없습니다.");
+    }
```

Comment:
```suggestion
    @Test
    @DisplayName("같은 이름이 여러 번 입력된 경우 예외 발생")
    void slitInputTest2() {
        assertThatThrownBy(() -> Application.splitInput("car1,car2,car1"))
                .isInstanceOf(IllegalArgumentException.class)
                .hasMessage("동일한 이름을 가진 차종을 입력하실 수 없습니다.");
    }
```

여긴 검증 잘 했네요 DisplayName만 손보면 될듯